### PR TITLE
style: update chain badges on wallet logos

### PR DESCRIPTION
**Pull request by dennyscode**
Ticket: https://lifi.atlassian.net/browse/LF-6399

### CODE REVIEW

**Review by chybisov — CHANGES_REQUESTED** [review]
image

Something is off with the avatar padding here. The icon should fill the whole circle to make sense for that huge badge. Also, we should consider reducing large badge border width. Wdyt? 🙂

File: src/components/Navbar/NavbarButtons/WalletManagementButtons.tsx [hunk 1]
```diff
@@ -140,15 +135,33 @@ export const WalletManagementButtons: React.FC<
...
       {isSuccess && activeChain ? (
-        <WalletMgmtAvatarContainer>
+        <Badge
+          overlap="circular"
+          className="badge"
+          sx={{
+            background: theme.palette.surface1.main,
+            borderRadius: '50%',
+            '& > span': { paddingLeft: theme.spacing(1.5) },
+          }}
```
[the remark this answers is not in the document]
Comment (dennyscode) [reply src/components/Navbar/NavbarButtons/WalletManagementButtons.tsx:145]: Good idea! Done

File: src/components/Avatar/Avatar.style.ts [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  background: theme.palette.background.paper,
+  borderRadius: '50%',
+}));
+
+export const LargeAvatar = styled(Avatar)(({ theme }) => ({
+  background: theme.palette.background.paper,
+  width: 48,
+  height: 48,
+  left: theme.spacing(0.5),
+  top: `-${theme.spacing(0.5)}`,
```
[the remark this answers is not in the document]
Comment (dennyscode) [reply src/components/Avatar/Avatar.style.ts:14]: 100 percent! Thanks

File: src/components/Avatar/Avatar.style.ts [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  border: `6px solid ${theme.palette.surface1.main}`,
+  width: 48,
+  height: 48,
+}));
+
+export const SmallAvatar = styled(Avatar)(({ theme }) => ({
+  background: theme.palette.background.paper,
+  width: '16px',
+  height: '16px',
+  top: '-2px',
```
[the remark this answers is not in the document]
Comment (dennyscode) [reply src/components/Avatar/Avatar.style.ts:32]: Good point! I have not used it theme.spacing for such small values (as `theme.spacing(-0.25)` for `-2px`), so far.
Generally, I try to use spacings specifically for offset values such as `paddings, margins` or positioning with `top, left, right, bottom`. But I don´t usually use it on `width` and `height`.

[the remark this answers is not in the document]
Comment (chybisov) [reply src/components/Avatar/Avatar.style.ts]: @dennyscode I think for width and height we should use numbers, not strings.

```
width: 16,
height: 16,
```

[the remark this answers is not in the document]
Comment (dennyscode) [reply src/components/Avatar/Avatar.style.ts]: For that, I have opened a new PR which does all that refactoring of spacings:
https://github.com/lifinance/jumper.exchange/pull/751

**Comment by dennyscode** [comment]
@chybisov On the Wallet-Icon: I have now decreased the padding to give more room for the wallet-icons. It still does not fill the whole circle, as you recommended - Trying it with MetaMask and Taho, coming to the conclusion that this approach does not properly cover both cases.
Because of that, I´d go with the following, but not extend it more:

image
image

**Comment by chybisov** [comment]
@dennyscode we should adjust the MM icon so it fits nicely in the circle with fox ears, not increase padding for all other icons that might have a background like Taho. 🙂

**Comment by dennyscode** [comment]
@chybisov I assume that the padding was supposed to be here by Design to give it that greyish background around.

Anyways, I gave it another try on the wallet logos. But I will check in with Vilen on it. But first of all, what do you think?
image
image

To compare: This is Figma
image

**Comment by chybisov** [comment]
@dennyscode, this is much better. We will improve on the size of MM icon so it fits nicely into the circle.

File: src/components/Menu/Menu.style.ts [hunk 4]
```diff
@@ -139,7 +139,7 @@ export const MenuPaper = styled(Paper, {
     maxHeight: 'calc( 100vh - 72px - 12px )',
     borderRadius: '12px !important',
     width: isWide ? '320px' : '288px',
-    marginTop: '-2px',
+    marginTop: `${theme.spacing(-0.25)}`,
```
Comment (chybisov) [inline src/components/Menu/Menu.style.ts:142]: No need to wrap it in string here

File: src/components/Navbar/NavbarButtons/WalletManagementButtons.tsx [hunk 5]
```diff
@@ -140,15 +136,28 @@ export const WalletManagementButtons: React.FC<
...
-            alt={`${activeChain.name}chain-logo`}
-          />
-        </WalletMgmtAvatarContainer>
+        </Badge>
       ) : null}
-      <Typography variant={'lifiBodyMediumStrong'} width={'auto'}>
+      <Typography
+        variant={'lifiBodyMediumStrong'}
+        width={'auto'}
+        sx={{ marginLeft: theme.spacing(1) }}
```
Comment (chybisov) [inline src/components/Navbar/NavbarButtons/WalletManagementButtons.tsx:159]: We can use numbers here, they will be automatically converted to the theme.spacing values.

File: src/components/Menu/Menu.style.ts (hunk 4 above)
Comment (dennyscode) [reply src/components/Menu/Menu.style.ts:142]: Typescript was complaining on a bunch of them, this was just my workaround. I will give this another try!

File: src/components/Navbar/NavbarButtons/WalletManagementButtons.tsx (hunk 5 above)
Comment (dennyscode) [reply src/components/Navbar/NavbarButtons/WalletManagementButtons.tsx:159]: I had no idea! Will give this a try

File: src/components/Menu/Menu.style.ts (hunk 4 above)
Comment (dennyscode) [reply src/components/Menu/Menu.style.ts]: My TS compiler seems to have done odd stuff throwing these errors. After a restart of vs code, it went fine - I corrected it here, and more is coming up in the spacings-refactoring PR.

**Comment by dennyscode** [comment]
image

**Review by chybisov — APPROVED** [review]

**Pull request closed without merging**